content/docs/08-troubleshooting/01-migration-guide/01-migration-guide-4-0.mdx

```diff
@@ -159,6 +159,14 @@ The following methods have been removed from the `streamText` result:
 - `pipeAIStreamToResponse`
 - `toAIStreamResponse`
 
+### Renamed "formatStreamPart" to "formatDataStreamPart"
+
+The `formatStreamPart` function has been renamed to `formatDataStreamPart`.
+
+### Renamed "parseStreamPart" to "parseDataStreamPart"
+
+The `parseStreamPart` function has been renamed to `parseDataStreamPart`.
+
```

Comment:
@shaper can you add a migration for this rename?

Reply:
Done in #3681 and #3682